src/Microsoft.Health.Dicom.Web/Program.cs

```diff
@@ -32,6 +32,7 @@ public static void Main(string[] args)
 
                     builder.AddDevelopmentAuthEnvironmentIfConfigured(builtConfig, "DicomServer");
                 })
+                .ConfigureKestrel(option => option.Limits.MaxRequestBodySize = uint.MaxValue)
```

Comment:
Can this be set to more conservative limit and only for POST on studies? Maybe 2GB. Can you also let Sneha know to updates these in the managed version of web proj.

Reply:
- I don't know if there is any way on Kestrel side for limit only on post, but our store service do has limitation on  MaxAllowedDicomFileSize.
- I'll update to 2GB